sys/vane/clay.hoon

```diff
@@ -1757,6 +1757,7 @@
     :*  hen  %pass
         [%foreign-x (scot %p our) (scot %p her) syd car (scot cas) pax]
         %f  %build  live=%.n  %pin
+        ::  XX perhaps should be now as in ++validate-plops
         (case-to-date cas)
```

Comment:
It does seem like this should also be now. (This comment also needs a trailing empty line.)